alluxio-ui/worker/src/constants/data/footerNavigation.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+import {INavigationData} from '@alluxio/common-ui/src/constants';
+
+export const footerNavigationData : INavigationData[] = [{
```

Comment:
Is this footer the same as the other one?

Reply:
I think yes, if so I'll move it to common.